types/index.d.ts

```diff
@@ -3,7 +3,7 @@
 // Definitions by: Rahim Alwer <https://github.com/mihar-22>
 
 import {queries, Queries, BoundFunction, EventType} from '@testing-library/dom'
-import { SvelteComponent, ComponentProps } from 'svelte/types/runtime'
+import { SvelteComponent, ComponentProps } from 'svelte'
```

Comment:
Hmmm... this will work with Svelte 4, but break if used with Svelte 3, right?

Reply:
@yanick  
My understanding, and from what I can tell, is that this works in both Svelte 3 and 4. 

The import path `'svelte/types/runtime'` is _only_ accessible in Svelte 3.

Reply:
Eeeexcellent!